### DISCUSSION TITLE
Doesn't work with Outline

### DISCUSSION THREAD

**Discussion by helmut72** [seaweedfs/seaweedfs#3712] [Q&A]
Hello,

I want to replace minio with seaweedfs to use it with outline [https://www.getoutline.com](https://www.getoutline.com). With minio every work, with seaweedfs uploading files fails. I've created a bucket `outline` and the logs in seaweedfs shows this:

```
I0918 06:42:47.452296 auth_credentials.go:229 post policy auth type
I0918 06:42:47.452381 s3api_object_handlers_postpolicy.go:28 PostPolicyBucketHandler outline
I0918 06:42:48.054437 filer.go:249 create directory: /buckets/outlineuploads/eaf10b3b-3b3a-4dcd-92a7-edf2a512276e/496e0040-27e0-4ea7-8d5a-e5e7ca37cbc6 drwxrwx--x
```

Looks like a `/` char is missing between outline and upload. Outline is configured to use path style `AWS_S3_FORCE_PATH_STYLE=true`.

Reading from the bucket works (I've cloned the existing files from minio bucket to seaweedfs with rclone):

```
I0918 06:53:38.437860 auth_credentials.go:225 v4 auth type
I0918 06:53:38.438223 auth_credentials.go:254 user name: Outline actions: [Read Write List Tagging Admin], action: Read
I0918 06:53:38.438319 s3api_object_handlers.go:165 GetObjectHandler outline /uploads/eaf10b3b-3b3a-4dcd-92a7-edf2a512276e/b8b645bd-ac95-4020-a8d2-2a63012c024d/macbook-air-spacegray-gallery2-20220606_GEO_DE.jpeg
I0918 06:53:38.438374 s3api_object_handlers.go:342 s3 proxying GET to http://172.27.0.2:8888/buckets/outline/uploads/eaf10b3b-3b3a-4dcd-92a7-edf2a512276e/b8b645bd-ac95-4020-a8d2-2a63012c024d/macbook-air-spacegray-gallery2-20220606_GEO_DE.jpeg
```

Now I'm a bit lost. I'm new to S3 and very new to seaweedfs. How can I debug where's really the problem. Is something wrong in outline or in seaweedfs? Are there some options to set in seaweedfs? I also can't see a file list in folders in seaweedfs with macos app `forklift` which works great with minio.

Thank you

**Comment by chrislusf** [comment]
Please create a GitHub issue with exact steps to reproduce the issue.

**Reply by helmut72** [reply]
https://github.com/seaweedfs/seaweedfs/issues/3737
